packages/fast-development-site-react/src/components/toc/toc-item.tsx

```diff
@@ -36,15 +36,24 @@ const style: ComponentStyles<ITocItemManagedClasses, IDevSiteDesignSystem> = {
         textDecoration: "none",
         display: "block",
         paddingLeft: toPx(48),
-        lineHeight: toPx(40),
-        outline: "0"
+        lineHeight: toPx(38),
+        border: `${toPx(1)} solid transparent`,
+        outline: "0",
+        "&:hover": {
+            backgroundColor: "rgba(0, 0, 0, 0.04)"
```

Comment:
Is there a way that we can use the design system here? When we setup theming of this, it will not change while other values will.

Reply:
Talked with @janechu @Falkicon supporting themeing on the Dev Site is not a priority right now and can be looked at later down the road. We shouldn't block this review

Reply:
Not blocking on this as I'm not required as a reviewer. Approved.